src/renderers/dom/ReactDOMNodeStreamRenderer.js

```diff
@@ -45,7 +45,7 @@ function renderToStream(element) {
   if (disableNewFiberFeatures) {
     invariant(
       React.isValidElement(element),
-      'renderToStream(): You must pass a valid ReactElement.',
+      'renderToStream(): Invalid component element.',
```

Comment:
Changed these for consistency with `ReactDOM.render` message. I'm not sure if it mattered in all cases (e.g. these ones are behind a flag that's now `false` in integration suite) but I changed them all for consistency.